cli/install.go

```diff
@@ -223,114 +278,85 @@ func (z *ZVM) InstallZls(version string) error {
 	}
 
 	arch, osType := zigStyleSysInfo()
+	expectedArchOs := fmt.Sprintf("%v-%v", arch, osType)
 
 	filename := "zls"
 	if osType == "windows" {
 		filename += ".exe"
 	}
 
 	// master does not need unzipping, zpm just serves full binary
-	var releaseUrl string
+	var shouldUnzip = true
 	if version == "master" {
-		releaseUrl = "https://api.github.com/repos/zigtools/zls/releases/latest"
-	} else {
-		// build url for tagged version
-		releaseUrl = fmt.Sprintf("https://api.github.com/repos/zigtools/zls/releases/tags/%v", version)
+		shouldUnzip = false
 	}
 
-	// get release information
-	resp, err := http.Get(releaseUrl)
+	downloadUrl, err := GetZlsDownloadUrl(version, expectedArchOs)
 	if err != nil {
+		fmt.Println("what?")
```

Comment:
Delete this. No random prints in ZVM.

Reply:
oh I totally left that in after being confused, oops!